cmd/osquery-perf/agent.go

```diff
@@ -142,8 +147,9 @@ type agent struct {
 }
 
 type entityCount struct {
-	common int
-	unique int
+	common     int
+	unique     int
+	vulnerable int
```

Comment:
It would be a bit more clear to define:
```go
type entityCount struct {
  common int
  unique int
}

type softwareEntityCount struct {
  entityCount
  vulnerable int
}
```